### PR TITLE
[SC-349] Use IERC20 instead of ERC20

### PR DESCRIPTION
**Pull request by barrutko**
Access to the `decimals` getter was the reason why there was `ERC20` used, instead of `IERC20` at first. Here's an alternative.

### CODE REVIEW

File: src/CapAutomator.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.13;
 
 import { Ownable } from "openzeppelin-contracts/access/Ownable.sol";
-import { ERC20 }   from "openzeppelin-contracts/token/ERC20/ERC20.sol";
+import { IERC20 }  from "openzeppelin-contracts/interfaces/IERC20.sol";
```
Comment (lucas-manuel) [inline src/CapAutomator.sol]: Should be able to use IERC20Detailed to get access to decimals

**Review by lucas-manuel — CHANGES_REQUESTED** [review]
One comment

File: src/CapAutomator.sol (hunk 1 above)
Comment (barrutko) [reply src/CapAutomator.sol]: Good call. Went for `IERC20Metadata`.

Comment (barrutko) [reply src/CapAutomator.sol]: Though imo it looks cleaner to use `IERC20` where possible and `IERC20Metadata` where necessary. But I can change it to use `IERC20Metadata` everywhere.

**Review by lucas-manuel — DISMISSED** [review]

**Review by hexonaut — DISMISSED** [review]
LGTM

**Pull request merged**